### PR TITLE
Added tlg0086.tlg015

### PR DESCRIPTION
**Pull request by planatheisa**
Checking Hook.

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +11 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4131 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg015/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3730)*

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +11 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4131 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg015/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3743)*

File: data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,618 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+    <teiHeader xml:lang="eng">
+        <fileDesc>
+            <titleStmt>
+                <title>De animalium motione</title>
```
Comment (sonofmun) [inline data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml]: Need @xml:lang="lat" attribute on title.

File: data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+                <listBibl>
+                    <biblStruct>
+                        <monogr>
+                            <title xml:lang="lat">Aristotelis De animalium motione et De animalium incessu</title>
+                            <editor>
+                                <persName>
+                                    <name xml:lang="lat">Vernerus Guilelmus Jaeger</name>
+                                </persName>
+                            </editor>
+                            <author ref="urn:cts:greekLit:tlg0086">Aristoteles</author>
```
Comment (sonofmun) [inline data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml]: This needs an @xml:lang attribute, probably "lat"

File: data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,618 @@
...
+                            <imprint>
+                                <publisher>Teubner</publisher>
+                                <pubPlace>Leipzig</pubPlace>
+                                <date>1913</date>
+                            </imprint>
+                            <biblScope unit="volume"/>
```
Comment (sonofmun) [inline data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml]: Since there is no volume number, I would just delete this `<biblScope>` element.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +1 |
| `metadata_total`   |       +1 |
| `nodes_count`      |      +11 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |    +4131 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0086/tlg015/__cts__.xml`                   | New      |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | New      |
| `./data/tlg0086/tlg015/__cts__.xml`                   | Passing  |
| `./data/tlg0086/tlg015/tlg0086.tlg015.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3796)*

**Pull request merged**